packages/ListBox/stories/Trigger.stories.js

```diff
@@ -1,94 +0,0 @@
-import React from "react";
-import { storiesOf } from "@storybook/react";
-import { getStoryName } from "storybook/storyTree";
-import Button from "@paprika/button";
-import RefreshIcon from "@paprika/icon/lib/Refresh";
-import ListBox from "../src";
-
-const storyName = getStoryName("ListBox");
-const options = ["Punisher", "Catwoman", "Venom", "Thunderbolts", "Deadpool", "Spawn", "Wolverine"];
-
-function renderOptions() {
-  return options.map(option => <ListBox.Option key={option}>{option}</ListBox.Option>);
-}
-
-const togglePopover = (dispatch, types) => () => {
-  dispatch({ type: types.togglePopover });
-};
-
-storiesOf(`${storyName}/Subcomponents/Trigger`, module)
```

Comment:
There's still a simple example in `Example/Trigger.js` and another example in the Docs page, but are you sure there's nothing good in this file that we're removing?

Reply:
That simple example is being used in the Docs page.

Reply:
I will expand on this more when i do the Docs page and explain more about the Trigger